internal/controller/config/config.go

```diff
@@ -26,21 +26,21 @@ type LogConfigurer interface {
 
 // MetricsConfigurer defines the accessor methods for metrics configurations.
 type MetricsConfigurer interface {
-	MetricsAddress() string
+	MetricsAddress() int
```

Comment:
Should the field be renamed to `Port`?

Reply:
Oh.. This was actually a bad refactoring as users wouldn't be able to provide the address anymore. 
Will revert.